docs/_static/js/cli-page.js

```diff
@@ -0,0 +1,25 @@
+(function () {
```

Comment:
**question:** Do we need this Javascript hack? How does it look without/with doing this?

Reply:
Unfortunately the due to the way that the sphinx-click package is implemented the generated CLI documentation page looks pretty terrible without it.

Reply:
Can you share a screenshot with and without it? Doesn't it just remove a header?

Reply:
It's not ideal, but also not entirely terrible imo. And adding Javascript to all pages feels pretty unsatisfying as well.

Apparently there were some efforts to [remove the header with spinx-click](https://github.com/click-contrib/sphinx-click/pull/100), but they have stalled unfortunately.

Can you use [`sphinx-click-process-description`](https://sphinx-click.readthedocs.io/en/latest/usage/#events) to remove the second Casanovo header that comes from the docstring? That should bring us pretty close to the desired layout already.

Reply:
I changed the implementation to use an event handler instead, here's what it looks like:

![image](https://github.com/Noble-Lab/casanovo/assets/16980750/4ef7f3c4-4f1c-4334-9a47-d7bffd8c8630)

Reply:
Yes, I think that this implementation is better. And the page doesn't look awful.